sys/parsoid.js

```diff
@@ -102,6 +102,10 @@ PSP._dependenciesUpdate = function(hyper, req) {
     var rp = req.params;
     var publicURI = '//' + rp.domain + '/api/rest_v1/page/html/' + encodeURIComponent(rp.title);
 
+    if (this.options.skip_updates) {
+        return P.resolve();
+    }
```

Comment:
Why do we need to duplicate the check in `parsoid.js` - we can just make it a no-op in `events.js` and don't change anything here - will simplify the config.